models/models.go

```diff
@@ -141,6 +142,7 @@ type DataPortalSeries struct {
 	Real                 *bool                   `json:"real,omitempty"`
 	SourceDescription    string                  `json:"source_description,omitempty"`
 	SourceLink           string                  `json:"source_link,omitempty"`
+	Indent               int                    `json:"indent,omitempty"`
```

Comment:
I guess this shows my still-far-to-go understanding of the biz logic, but here you have made the indent a property of a series/measurement, whereas in my udaman implementation, it is only at that indent relative to a data list. I guess there's a simple explanation for this, something to do with assumptions built-in to the data portal, so I'll just approve the PR and wait to hear it from you :=)